committeeoversightapp/management/commands/import_data.py

```diff
@@ -0,0 +1,210 @@
+import os
+import csv
+
+from datetime import datetime
+from django.core.management.base import BaseCommand, CommandError
+from django.core.exceptions import ObjectDoesNotExist
+from django.conf import settings
+from django.db import IntegrityError
+from django.db.models import Q
+
+from opencivicdata.core.models import Organization, OrganizationName
+from opencivicdata.legislative.models import Event, EventSource, EventParticipant
+from committeeoversightapp.models import HearingCategory, Committee
+
+jurisdiction_id = 'ocd-jurisdiction/country:us/legislature'
+bad_rows = []
+
+def new_committee(committee_key, event, row_count):
```

Comment:
Is there a reason to have this function outside of the Command class?

Reply:
Hmmm I was getting a `NameError` when I had it inside. Do you know why?

Reply:
Are you calling it with the `self` keyword?

```python
# declare function
def get_committee(self, committee_name, parent):
......

# call function
self.get_committee(committee_name, parent)
```

I was able to do this locally. Let me know if you need to talk through it.